llvm/test/Transforms/LoopVectorize/vector-to-scalar-cast.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: opt --force-widen-divrem-via-safe-divisor=false -passes=loop-vectorize -S --debug-only=loop-vectorize < %s 2>&1 | FileCheck %s
+
+; CHECK: LV: Not vectorizing: Found unvectorizable type   %s2 = bitcast <2 x i16> %vec1 to i32
+; CHECK-NOT: Assertion
```

Comment:
Assertion? Is there any debug string that outputs assertion?

Reply:
The test triggers an assert without the code I added in the legalizer, so I just added this to make sure no assert is triggered any longer. Maybe it's not necessary and the test would fail anyway if we trigger an assert? The debug output will print some of these statements before the assert triggers so that's why I was a bit concerned about having only the CHECK.

Should I remove this?

Reply:
Yes, the test would fail anyway. No need to check-not assertion.

Reply:
I have rewritten the checks, let me know what you think.